specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/expressRoutePort.json

```diff
@@ -913,6 +1159,13 @@
         "identity": {
           "$ref": "./network.json#/definitions/ManagedServiceIdentity",
           "description": "The identity of ExpressRoutePort, if configured."
+        },
+        "authorizations": {
```

Comment:
This shouldn't be here either.  A child resource should not appear in any way inside the parents request/response.  You have separate APIs that should be used to interact with authorizations

Reply:
Got it. Thanks.